18 Normalizr/src/model/api/student.ts

```diff
@@ -3,12 +3,14 @@ export class Student {
   gotActiveTraining: boolean;
   fullname: string;
   email: string;
+  countryId: number;
 
   public constructor() {
     this.id = -1;
     this.gotActiveTraining = false;
     this.fullname = "";
     this.email = "";
+    this.countryId = -1;
```

Comment:
I think we are using this value because [Uncontrolled input warning](https://github.com/erikras/redux-form/issues/735) (If you initialize input value to null, it means that is uncontrolled component) But I think that this initialization has to be component liability and it doesn't delegate to class constructor used in reducer default value state. What do you think?